lib/camel_snake_struct.rb

```diff
@@ -30,7 +30,7 @@ def self.walk_example(new_example)
 
   def self.store_meta_data(example_class, m_name, result)
     types_meta_data = (example_class.types_meta_data[m_name] ||= Type__Meta__Data.new(Set.new, false))
-    if result.is_a?(Array)
+    if result.is_a?(Array) && result.length > 0
```

Comment:
I think we actual want to fix the tapioca compiler.
https://github.com/NEXL-LTS/nexl360/blob/main/sorbet/tapioca/compilers/camel_snake_struct_compiler.rb#L27

maybe something like:
```ruby
  classes.size > 0 "T.any(#{classes.join(', ')})" || "T.untyped"
```

Reply:
Nice..will look in..

Another thing I realised that is we don't add the method to the set if array is empty. Do we even need tapioca to generate types?

Reply:
yes, can't add the method if we don't know what type goes into it.

Reply:
Hi @grantspeelman  

So, added a check for that we dont add type if array is empty